src/Parser/Mage/Arcane/Modules/ManaChart/Mana.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import TabComponent from './TabComponent';
+
+class ManaTab extends Analyzer {
+
+  tab() {
+    return {
+      title: 'Mana',
+      url: 'mana',
+      render: () => (
```

Comment:
for a simple function+component like this, what about putting it on one line instead, eg

```jsx
render: () => <TabComponent parser={this.owner} />,
```